### DISCUSSION TITLE
Incorrect Campus Information

### DISCUSSION THREAD

**Discussion by CavLydia** [community/community#69446] [GitHub Education]
### Hi everyone,

I am a current student at Northeastern University's Toronto campus. I am attempting to verify my student status on GitHub Education, but the system is reporting that our school does not have a campus in my region. However, the fact is that NEU has two campuses in Canada, one in Toronto and one in Vancouver, and they have been in existence for several years. Due to this error, I am unable to complete the verification process on my own

**Comment by mantasu** [comment]
Perhaps there is a more specific university name? E.g., "Northeastern University, Toronto Campus".
